packages/design-picker/src/utils/available-designs.ts

```diff
@@ -57,31 +56,18 @@ export const mShotOptions = (): MShotsOptions => {
 const canUseWebP = getCanUseWebP();
 
 export const getDesignImageUrl = ( design: Design ): string => {
```

Comment:
I know it's not really the scope of this PR but since we're extracting this function, let's try to make more clear what should be updated in order for the cache busting to happen.

```suggestion
// Bump the version query param here to cache bust the images after running bin/generate-gutenboarding-design-thumbnails.js
export const getDesignImageUrl = ( design: Design ): string => {
```